<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilePathEditor</class>
 <widget class="QWidget" name="FilePathEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>31</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="pathEdit"/>
   </item>
   <item>
    <widget class="QToolButton" name="browseBtn">
     <property name="text">
      <string notr="true">...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="editBtn">
     <property name="text">
      <string>Edit...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pathEdit</tabstop>
  <tabstop>browseBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
